src/Models/UserModel.php

```diff
@@ -114,6 +116,7 @@ public function addToDefaultGroup(User $user)
     public function fake(Generator &$faker)
     {
         return [
+            'id'       => $faker->randomNumber(5, true),
```

Comment:
I assume you added this to generate "valid" User instances but it will prevent auto-increment from working and could create collision if tests make multiple accounts.

Reply:
Ah, it is used to create records in the database?

I added it, because test failed.

Reply:
```
There was 1 failure:

1) Tests\Authentication\AuthHelperTest::testUserIdReturnsId
User ID must not be null. in /Users/kenji/work/codeigniter/codeigniter-shield/src/Authentication/Traits/Authenticatable.php:31
```

Reply:
Yes, much better! Thanks.

Reply:
Note that another option is to use `fake()`'s third parameter to `$persist` the object into the database, which will assign it an ID: https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/Helpers/test_helper.php